src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -634,12 +634,10 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
         };
 
         auto eltwise_supports_fusings = [&](eltwise_node& node) -> bool {
-            if (_lo.get_optimization_attributes().use_onednn_impls == 0) {
-                auto out_layout = node.get_output_layout();
-                if (out_layout.data_type == data_types::f16 && out_layout.batch() > 1 &&
-                    (_lo.get_optimization_attributes().fs_b_yx_fsv32_network || out_layout.format == format::fs_b_yx_fsv32)) {
-                    return false;
-                }
+            auto out_layout = node.get_output_layout();
```

Comment:
Do we have a test that covers this scenario?

Reply:
Fusion test 'fusings_gpu/eltwise_fp32_fsv32.add/0' fails with "Best_kernel.empty() should false..." error because it forces implementation to fs_b_yx_fsv32. At master branch graph optimizer, if onednn is enabled then all eltwise fusion allowed but that cldnn kernel doesn't support fusion and it makes test failures.